docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to provision a cluster, deploy Kyma, and <!--- update is not among the alpha commands ---> with alpha commands.
+
+## Provision a cluster locally or using cloud providers
+<!---provision gke is not among the alpha commands--->
+To provision a cluster on a specific cloud provider (in this example GCP), run:
+
+```bash
+kyma alpha provision gke --credentials {SERVICE_ACCOUNT_KEY_FILE_PATH} --name {CLUSTER_NAME} --project {GCP_PROJECT} 
```

Comment:
There is only one new provision command in alpha. That is k3s. So the commands listed here will not even work..

Reply:
Alright, I threw out gke and minikube and added k3s instead.
Didn't find `--credentials` and `--project` among the alpha provision flags though, and none of the existing flags is an easy match. So I'm gonna assume that k3s doesn't require a service account key or a project. 

```suggestion
kyma alpha provision k3s --name {CLUSTER_NAME} 
```